www/src/components/Menu.scss

```diff
@@ -6,14 +6,14 @@
   padding: $spacer .625rem 2rem;
   border-right: $border-width solid $light-400;
 
-  &-title {
+  .pgn-doc__menu-title {
```

Comment:
These changes are necessary to comply with the general principles for writing classes in the OpenEdx ecosystem. I noticed that the full name of the class is almost always written to simplify searching in the code